composer.json

```diff
@@ -23,7 +23,8 @@
         "docs": "https://portphp.readthedocs.org"
     },
     "require": {
-        "php": ">=5.4.0"
+        "php": ">=5.4.0",
+        "portphp/portphp": "dev-master"
```

Comment:
On packagist :stuck_out_tongue_winking_eye: 

`"portphp/portphp": "1.0.*@dev"` (under the master branch)